articles/search/search-filters.md

```diff
@@ -67,23 +66,23 @@ The following examples represent prototypical filter definitions in several APIs
 
 ```http
 # Option 1:  Use $filter for GET
-GET https://[service name].search.windows.net/indexes/hotels/docs?search=*&$filter=baseRate lt 150&$select=hotelId,description&api-version=2019-05-06
+GET https://[service name].search.windows.net/indexes/hotels/docs?search=*&$filter=Rooms/any(room: room/BaseRate lt 150.0)&$select=HotelId, HotelName, Rooms/Description, Rooms/BaseRate
```

Comment:
This won't work without api-version.

Reply:
Thanks @brjohnstmsft : agree! Since on search explorer it gets added by default , I had pasted the search query only. Updated it now